core/domain/skill_domain.py

```diff
@@ -551,6 +567,23 @@ def update_language_code(self, language_code):
         """
         self.language_code = language_code
 
+    def update_superseding_skill_id(self, superseding_skill_id):
+        """Updates the superseding skill id of the skill.
+
+        Args:
+            superseding_skill_id: str. Id of the skill that supersedes this one.
+        """
+        self.superseding_skill_id = superseding_skill_id
+
+    def update_all_questions_merged(self, all_questions_merged):
```

Comment:
Perhaps better to say "record_that_all_questions_are_merged" since this is a one-way action?

Reply:
Updated to 'record_that_all_questions_are_merged'